python/eos/analysis_file.py

```diff
@@ -23,10 +23,12 @@
 import yaml
 import inspect
 from collections import Counter
+from collections import OrderedDict
```

Comment:
Since python 3.7, standard dictionaries preserve insertion order, so `OrderedDict` not needed